spring-integration-azure/src/main/java/com/microsoft/azure/spring/integration/eventhub/DefaultEventHubClientFactory.java

```diff
@@ -34,30 +37,25 @@
  */
 public class DefaultEventHubClientFactory implements EventHubClientFactory, DisposableBean {
     private static final Log LOGGER = LogFactory.getLog(DefaultEventHubClientFactory.class);
-    // eventHubName -> eventHubClient
-    private final ConcurrentHashMap<String, EventHubClient> clientMap = new ConcurrentHashMap<>();
 
-    // eventHubName -> connectionString
-    private final ConcurrentHashMap<String, String> connectionStringMap = new ConcurrentHashMap<>();
+    private final Map<String, EventHubClient> clientsByName = new ConcurrentHashMap<>();
```

Comment:
it's a 1 to 1 mapping so i guess clientByName better?

Reply:
Yes.